bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/item/ItemResource.java

```diff
@@ -539,7 +546,8 @@ public Response createOrUpdateItem(
     @ApiOperation(value = "Adds a list of items to the registry or updates the existing items.")
     @ApiResponses(value = { @ApiResponse(code = 200, message = "OK", response = String.class),
             @ApiResponse(code = 400, message = "Item list is null.") })
-    public Response createOrUpdateItems(@ApiParam(value = "array of item data", required = true) GroupItemDTO[] items) {
+    public Response createOrUpdateItems(
+            @ApiParam(value = "array of item data", required = true) GroupItemDTO @Nullable [] items) {
```

Comment:
Again, `required = true` and `@Nullable` are contradicting each other, aren't they?